pkg/deploy/util/util.go

```diff
@@ -292,7 +311,8 @@ func MakeDeployment(config *deployapi.DeploymentConfig, codec runtime.Codec) (*a
 				deployapi.DesiredReplicasAnnotation:    strconv.Itoa(int(config.Spec.Replicas)),
 				deployapi.DeploymentReplicasAnnotation: strconv.Itoa(0),
 			},
-			Labels: controllerLabels,
+			Labels:          controllerLabels,
+			OwnerReferences: []metav1.OwnerReference{*NewControllerRef(config)},
```

Comment:
I don't think there is anything wrong with dereferencing a function result, but I will extract it into variable.